prometheus/scylla_servers.yml

```diff
@@ -1,4 +1 @@
-# List Scylla end points
-
-- targets:
-  - 172.17.0.2:9180
\ No newline at end of file
+  - targets: ["172.17.0.2:9180"]
```

Comment:
This commit changes scylla_servers.yml, not dashboard definition

Reply:
Thanks, obviously a mistke